File: YaContests/SummerSchool2025Testirovanye/findOshibka.js
function findLargestNumber(numbers) {
  if (numbers.length === 0) return undefined; // или можно вернуть null

  let largest = numbers[0]; // инициализируем первым элементом

  for (let i = 0; i < numbers.length; i++) {
    if (numbers[i] > largest) {
      largest = numbers[i];
    }
  }

  return largest;
}

function main() {
  const fs = require("fs");
  const input = fs.readFileSync(0, "utf8").trim();

  if (input === "") {
    // обработка пустого ввода
    console.log(""); // или можно вернуть undefined/null
    return;
  }

  const numbers = input.split(" ").map(Number);

  console.log(findLargestNumber(numbers));
}

main();

File: contests/yandex/SummerSchool2025Testirovanye/findOshibka.py
def find_largest_number(numbers):
    if not numbers: # Обработка пустого списка
        return None

    largest = numbers[0]  # инициализируем первым элементом

    for num in numbers:
        if num > largest:
            largest = num

    return largest

def main():
    import sys
    input_data = sys.stdin.read().strip()

    if not input_data:  # если ввод пустой
        print("")  # или можно вернуть None или другое значение по умолчанию
        return

    numbers = list(map(int, input_data.split()))

    largest = find_largest_number(numbers)
    print(largest)

if __name__ == "__main__":
    main()
